object.js

```diff
@@ -0,0 +1,125 @@
+// Создайте объект calculator (калькулятор) с тремя методами:
+
+// read() (читать) запрашивает два значения и сохраняет их как свойства объекта.
+// sum() (суммировать) возвращает сумму сохранённых значений.
+// mul() (умножить) перемножает сохранённые значения и возвращает результат.
+
+let calculator = {
+    read() {
+        this.a = +prompt("Введите значение а");
```

Comment:
what if I enter 'abc'?

Reply:
it will be Nan

Reply:
i know))))
you should avert using NaN further on
check the value to be true

Reply:
Maybe we need to add the condition with if(this.a && this b){ return this.a*this.b}
The result will be underfined

Reply:
check if it's a number then do smth 
If not - add zero value to the variable

Reply:
edited